### ISSUE TITLE
Wales, Scotland and England showing up as Black Flag

### ISSUE THREAD

**Issue by didiersurka** [idkhow2type/Flagmoji#1]
By emojipedia: 

> The Flag: Wales emoji is a [tag sequence](https://emojipedia.org/emoji-tag-sequence) combining [🏴 Black Flag](https://emojipedia.org/black-flag), [󠁧 Tag Latin Small Letter G](https://emojipedia.org/tag-latin-small-letter-g), [󠁢 Tag Latin Small Letter B](https://emojipedia.org/tag-latin-small-letter-b), [󠁷 Tag Latin Small Letter W](https://emojipedia.org/tag-latin-small-letter-w), [󠁬 Tag Latin Small Letter L](https://emojipedia.org/tag-latin-small-letter-l), [󠁳 Tag Latin Small Letter S](https://emojipedia.org/tag-latin-small-letter-s) and [󠁿 Cancel Tag](https://emojipedia.org/cancel-tag). These display as a single emoji on supported platforms.

Seems it's a compound emoji, could this be the reason?

**Comment by idkhow2type** [comment]
Yep, I did know about this but never got around to fix it because the emoji parsing was really crude, will work on it!
